playbooks/common/openshift-node/config.yml

```diff
@@ -27,10 +27,12 @@
     stat:
       path: "{{ item }}"
     with_items:
-    - "/etc/openshift/node/node.key"
-    - "/etc/openshift/node/node.kubeconfig"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.crt"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.key"
+    - "/etc/openshift/node/system:node:{{ openshift.common.hostname }}.kubeconfig"
```

Comment:
I prefer the output of create-node-config, since it doesn't include a hostname in the filenames, which seems fragile. If create-api-client-config must be used, I'd like to allow the basename to be specifiable via arg to allow these files to stay named `node.kubeconfig`, etc